### PR TITLE
some-improvements

### PR DESCRIPTION
**Pull request by wissog**

### CODE REVIEW

**Comment by wissog** [comment]
Will be used

**Comment by wissog** [comment]
It seems that rebase happened incorrectly and i cannot change it

File: src/components/Sections/Likutim/Likut.js [hunk 1]
```diff
@@ -0,0 +1,187 @@
+import React, { useEffect, useState, useContext, useRef } from 'react';
```
[the remark this answers is not in the document]
Comment (wissog) [reply src/components/Sections/Likutim/Likut.js:1]: this is from another PR of likutim

File: public/locales/ru/common.json [hunk 2]
```diff
@@ -459,7 +459,7 @@
       "same-collection": "Рекомендуемые передачи из",
       "any-topic": "Любая тема",
       "default": "Рекомендуемые",
-      "popular": "",
+      "popular": "Наиболее просматриваемые",
```
Comment (wissog) [inline public/locales/ru/common.json:462]: not mine, merged from before

Comment (gshilin) [reply public/locales/ru/common.json:462]: @bbfsdev Should it be empty or not?

**Comment by wissog** [comment]
it seems the label was forgotten, so someone added it...
בברכה,
גבריאל ויסוצקי
נייד: 054-5606-798

File: public/locales/ru/common.json (hunk 2 above)
Comment (bbfsdev) [reply public/locales/ru/common.json:462]: Mine should be empty, please duplicate this key... Make your own.

**Comment by wissog** [comment]
i don't need it,

cannot understand how it has got into my branch, i didn't make any changes
there...
בברכה,
גבריאל ויסוצקי
נייד: 054-5606-798

File: src/stylesheets/includes/_not-sorted-scss.scss [hunk 3]
```diff
@@ -1442,6 +1442,8 @@
 
 .ui.flowing.popup {
   z-index: 10000;
+  margin: 0 !important;
```
Comment (gshilin) [inline src/stylesheets/includes/_not-sorted-scss.scss:1445]: This will change **all** popups. Are you sure?

Comment (wissog) [reply src/stylesheets/includes/_not-sorted-scss.scss:1444]: yea, filter popups are fine

**Review by gshilin — APPROVED** [review]

**Pull request merged**